Bottom: 2320800b501a6466c5d367bf27d76de089454cc9
Top:    c0762fd9e1a128f21e1a52a24375b3af06fcd4dc
Author: Igor Stoppa <igor.stoppa@huawei.com>
Date:   2018-05-03 15:38:00 +0400

Refresh of preliminary-self-test-for

---

diff --git a/mm/Kconfig.debug b/mm/Kconfig.debug
index 15ecaa2ae96a..dd6ffbe361d7 100644
--- a/mm/Kconfig.debug
+++ b/mm/Kconfig.debug
@@ -95,12 +95,13 @@ config DEBUG_RODATA_TEST
     ---help---
       This option enables a testcase for the setting rodata read-only.
 
+
 config TEST_PROTECTABLE_MEMORY
-	tristate "Run self test for pmalloc memory allocator"
+       tristate "Run self test for pmalloc memory allocator"
         depends on MMU
-	depends on ARCH_HAS_SET_MEMORY
-	select PROTECTABLE_MEMORY
-	default n
-	help
-	  Tries to verify that pmalloc works correctly and that the memory
-	  is effectively protected.
+       depends on ARCH_HAS_SET_MEMORY
+       select PROTECTABLE_MEMORY
+       default n
+       help
+         Tries to verify that pmalloc works correctly and that the memory
+         is effectively protected.
